### PR TITLE
Navigating to latest deployed config from subnav

### PR DESCRIPTION
**Pull request by jgruica**
This change is updating config tab to go to the deployed config instead of latest.

### CODE REVIEW

**Review by GraysonNull — APPROVED** [review]

File: web/src/components/shared/SubNavBar/SubNavBar.jsx [hunk 1]
```diff
@@ -14,6 +14,7 @@ export default function SubNavBar(props) {
     slug = `helm/${app.id}`;
   }
   const kotsSequence = app.currentSequence;
+  const latestDeployedSequence = app?.downstreams[0]?.currentVersion?.parentSequence;
```
Comment (salahalsaleh) [inline web/src/components/shared/SubNavBar/SubNavBar.jsx:17]:
```suggestion
  const currentDeployedSequence = app?.downstreams[0]?.currentVersion?.parentSequence;
```

**Review by salahalsaleh — COMMENTED** [review]
What happens if there's no version deployed?

File: web/src/components/apps/AppConfig.jsx [hunk 2]
```diff
@@ -176,7 +176,8 @@ class AppConfig extends Component {
     if (match.params.sequence != undefined) {
       return parseInt(match.params.sequence);
     }
-    return app?.currentSequence;
+
+    return app?.downstreams[0]?.currentVersion?.parentSequence || app?.currentSequence;
```
Comment (salahalsaleh) [inline web/src/components/apps/AppConfig.jsx:180]: what if the `parentSequence` is `0`? like the currently deployed version is sequence `0` :)

File: web/src/components/shared/SubNavBar/SubNavBar.jsx [hunk 3]
```diff
@@ -14,6 +14,7 @@ export default function SubNavBar(props) {
     slug = `helm/${app.id}`;
   }
   const kotsSequence = app.currentSequence;
+  const configSequence = app?.downstreams[0]?.currentVersion?.parentSequence || app?.currentSequence;
```
Comment (salahalsaleh) [inline web/src/components/shared/SubNavBar/SubNavBar.jsx:17]: same here

**Review by salahalsaleh — APPROVED** [review]

**Pull request merged**